scss/footer.scss

```diff
@@ -0,0 +1,24 @@
+.footer-page {
+  align-items: center;  
+  background-color: $blue;
+  display: flex;
+  flex-direction: column;
+    
+  .footer-logo {
+    height: 50px;
+    margin: 10px;
+  }
+  
+  .footer-title {
+    color: $white;
+    font-size: $text-medium;
+    margin: 10px;
+  }
+
+  .footer-text {
```

Comment:
It's not necessary nesting this class definition, please move it outside of here